### ISSUE TITLE
No member completion for empty prefix

### ISSUE THREAD

**Issue by mniewrzal** [felixfbecker/php-language-server#195]
No completion for two separate cases.
```php
<?php
class TestClass 
{
    function abc(){
        $this->
    }
}
```
```php
TestClass::
```
If at least one letter will be added then completion will work.

**Comment by felixfbecker** [comment]
What about these test cases?
https://github.com/felixfbecker/php-language-server/blob/master/fixtures/completion/static.php
https://github.com/felixfbecker/php-language-server/blob/master/fixtures/completion/property.php

**Comment by mniewrzal** [comment]
You are right. Only case with `$this` is broken (I didn't have latest version of php-parser).

**Comment by felixfbecker** [comment]
Still happening?
